src/client/Microsoft.Identity.Client/ApiConfig/Parameters/AbstractAcquireTokenConfidentialClientParameters.cs

```diff
@@ -20,5 +20,10 @@ internal abstract class AbstractAcquireTokenConfidentialClientParameters
         /// This overrides application config settings.
         /// </summary>
         public bool? SendX5C { get; set; }
+
+        /// <summary>
+        /// Whether to use MTLS Proof of Possession (PoP)
+        /// </summary>
+        public bool UseMtlsPop { get; set; } = false;

```

Comment:
We don't plan to extend this to web app and web api any time soon. So I recommend you move this to client_credential paramters